fastlane/lib/fastlane/actions/upload_to_play_store.rb

```diff
@@ -16,6 +16,11 @@ def self.run(params)
           end
         end
 
+        # If no AAB param was provided, try to fill in the value from lane context.
+        if params[:aab].nil?
```

Comment:
I think we might want something like

```ruby
if params[:aab].nil? && Actions.lane_context[SharedValues::GRADLE_ALL_AAB_OUTPUT_PATHS].count == 1
```

to avoid doing something strange for users with multiple build variants?

Alternatively, we could avoid setting those here if there are `>1` variants

https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/gradle.rb#L80-L81

CC @joshdholtz

Reply:
Yeah, I'm not sure about the perfect solution.
Waiting on @joshdholtz 's input then

Reply:
I think @getaaron right that we need to use `SharedValues::GRADLE_ALL_AAB_OUTPUT_PATHS` but I think the logic should probably be something like...
```rb
# If no AAB param was provided, try to fill in the value from lane context.
# First GRADLE_ALL_AAB_OUTPUT_PATHS if only one
# Else from GRADLE_AAB_OUTPUT_PATH
if params[:aab].nil?
  all_aab_paths = Actions.lane_context[SharedValues::GRADLE_ALL_AAB_OUTPUT_PATHS] || []
  if all_aab_paths.count == 1
    params[:aab] = all_aab_paths.first
  else
    params[:aab] = Actions.lane_context[SharedValues::GRADLE_AAB_OUTPUT_PATH]
  end
end
```

This way we are making use of both `GRADLE_ALL_AAB_OUTPUT_PATHS` and `GRADLE_AAB_OUTPUT_PATH`. Even though the odds of these two defaults being different is probably super super super small, I think this is the most thorough approach. Thoughts?

Reply:
I'll update and ping you when it's done.